extension/src/extension.ts

```diff
@@ -393,11 +393,14 @@ export class Extension extends Disposable implements IExtension {
         this.dvcRoots,
         this.updatesPaused,
         this.resourceLocator
-      ),
-      this.plots.create(this.dvcRoots, this.updatesPaused, this.resourceLocator)
+      )
     ])
-
-    this.experiments.linkRepositories(this.plots)
+    this.plots.create(
+      this.dvcRoots,
+      this.updatesPaused,
+      this.resourceLocator,
+      this.experiments
```

Comment:
[F] By creating this after we have created the experiments repositories we can simply/remove a lot of the optional chaining logic from `Plots` and `PlotsData` (model is passed in on instantiation instead of being set afterwards).